torchao/quantization/quant_api.py

```diff
@@ -219,418 +219,418 @@ def replace_conv2d_1x1(conv):
         lm_eval,
         MultiInput,
     )
+else:
+    print("lm_eval not available, skip defining GPTQQuantizer")
+
+
+class GPTQQuantizer(Quantizer):
```

Comment:
should this live under `if` branch as well?

Reply:
right now GPTQQuantizer has dependency on lm-eval though, it's not for accuracy eval, it's used to gather the inputs from tasks. not sure how we want to remove the dependency there